server/src/main/java/org/elasticsearch/action/update/TransportUpdateAction.java

```diff
@@ -185,7 +184,7 @@ protected void shardOperation(final UpdateRequest request, final ActionListener<
         final UpdateHelper.Result result = updateHelper.prepare(request, indexShard, threadPool::absoluteTimeInMillis);
         switch (result.getResponseResult()) {
             case CREATED -> {
-                IndexRequest upsertRequest = removeInferenceMetadataField(indexService, result.action());
```

Comment:
Unfortunately we still need to do something. If the semantic text field does not hold any value, we need to remove it from the get request since it is always computed from copy_to fields.